File: 0069_Sqrt(x)/Sqrt(x).cs
public class Solution
{
    public int MySqrt(int x)
    {
        if (x <= 1) return x;
        for(long i = 0; i <= x; i++)
        {
            if (i * i > x) return (int)i - 1;
        }
        
        return -1;       
    }
}

File: 0069_Sqrt(x)/Sqrt(x).java
class Solution {
    public int mySqrt(int x) {
        if(x <= 1) return x;
        int l = 1, r = x/2 + 1;
        while(l < r) {
            int m = (r-l)/2 + l;
            if(m > x/m) {
                r = m;
            }else{
                l = m+1;
            }
        }
        
        return l-1;
    }
}